Cargo.toml

```diff
@@ -12,6 +12,9 @@ rand_distr = "0.4"
 num-traits = "0.2"
 ndarray = { git = "https://github.com/LazaroHurtado/ndarray", branch = "windows_stride_feature" }
 ordered-float = "1.0.1"
+serde = "1.0"
+serde-pickle = "0.5.0"
```

Comment:
How come we are using `serde-pickle` version 0.5 over [1.1](https://docs.rs/serde-pickle/latest/serde_pickle/index.html)?

Reply:
Leftover from Demba's PR, will change.